pkgs/top-level/python-packages.nix

```diff
@@ -17329,6 +17353,29 @@ in {
     };
   };
 
+  unittest-xml-reporting = buildPythonPackage rec {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.